k8s/postgres.storage.yml

```diff
@@ -0,0 +1,29 @@
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: postgres-pv-volume
+  labels:
+    type: local
+    app: postgresdb
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteMany
```

Comment:
This access mode may not be supported on all kubernetes deployments, see https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes. You probably want `ReadWriteOnce` since you only have a single postgres replica (and even with multiple replicas you can't share a single PVC).